SourceX/controls/menu_controls.h

```diff
@@ -21,4 +21,7 @@ enum MenuAction {
 
 MenuAction GetMenuAction(const SDL_Event &event);
 
+// Menu action from holding the left stick or DPad.

```

Comment:
Use Doxygen style for documentation
```suggestion
/** Menu action from holding the left stick or DPad. */
```